pkg/version/version.go

```diff
@@ -1,4 +1,4 @@
 package version
 
-var VERSION = "6.1.4"
+var VERSION = "6.1.5"
```

Comment:
Please undo this change, the version bump is done at release time.

Reply:
done